Demo/Viper-v2-Demo/Viper-v2-Demo/Modules/Details/PokemonDetailsInterfaces.swift

```diff
@@ -18,9 +18,15 @@ protocol PokemonDetailsWireframeInterface: WireframeInterface {
 }
 
 protocol PokemonDetailsViewInterface: ViewInterface {
+    func setViewTitle(_ title: String?)
+    func setHeaderImage(with url: URL?)
+    func reloadData()
 }
 
 protocol PokemonDetailsPresenterInterface: PresenterInterface {
+    func numberOfSections() -> Int
```

Comment:
It might be easier to just expose the array of sections to view rather then writing 3 separate for the same thing.